### ISSUE TITLE
Install V2.2 and get parent_id is not a valid integer

### ISSUE THREAD

**Issue by gzh1960** [sgladstone/com.fountaintribe.hebrewcalendarhelper#1]
Hi Sarah,
Have 4.6.9 CiviCRM. Tried to install V2.2 of hebrewcalendarhelper extension and I get the following error 'parent_id is not a valid integer'.

[hebrewhelperinstall.txt](https://github.com/sgladstone/com.fountaintribe.hebrewcalendarhelper/files/428148/hebrewhelperinstall.txt)

Any ideas please?
Thanks Garry

**Comment by sgladstone** [comment]
This is fixed in version 2.5.   (It was due to a difference between the CiviCRM core API from 4.6.x to 4.7.x)

**Comment by gzh1960** [comment]
Thanks Sarah, it has resolved the issue but fyi only after I installed twice. See the attached log (**1st half**) for the messages. I reinstalled a 2nd time and it woke up and was fine.

The **2nd half of the attached log** shows another issue related to running the Custom Search:
Configuration Issue: Unrecognized System date format: d M yy
I finished up changing the date format to the American format (mm/dd/yyyy) to get the search to run but would really like it to work with Australian format of dd/mm/yyyy. Any thoughts here please?

[hebrewhelperinstall2.txt](https://github.com/sgladstone/com.fountaintribe.hebrewcalendarhelper/files/429130/hebrewhelperinstall2.txt)

**Comment by sgladstone** [comment]
If you do a fresh install of version 2.6, do you have any issues or warnings, etc?

BTW: version 2.6 can deal with the date format 'd M yy'

**Comment by sgladstone** [comment]
If you do a fresh install of version 2.7, do you have any issues or warnings, etc?

BTW: version 2.7 can deal with the date format 'd M yy' or any other date format.

**Comment by gzh1960** [comment]
Disabled 2.5 and Installed 2.7.
Changed localisation date format to d M yyyy and ran Call AllHebrewDates.Calculate API job. 
Error as follows:
Configuration Issue: Unrecognized System date format: d M yy

So changed format to dd/mm/yyyy and reran job and all clear no errors.
